src/main/java/frc/robot/constants/ArmPositions.java

```diff
@@ -3,8 +3,8 @@
 public enum ArmPositions {
     HANDOFF_AND_DEFAULT_SHOT(0, 0.435),
     AMP(20, 0.69),
-    TRAP_SEGUEAY(0, 0),
-    TRAP(44, 0),
+    TRAP_SEGUEAY(44.8, 0.39),
+    TRAP(44.8, 0.69),
     SPEAKER_SHOT(0, 0.43);
 
     public double elevator, pivot;
```

Comment:
these should also include what units these positions are in. for example `elevatorHeightMeters` and `pivotAngleDeg` or something like that